diff -r 6625feb3fc7f config_unix.py
--- config_unix.py	Mon Aug 13 07:39:41 2012 -0500
+++ config_unix.py	Thu Sep 06 09:52:21 2012 +1000
@@ -198,16 +198,15 @@
         print ('Unable to run "sdl-config". Please make sure a development version of SDL is installed.')
         raise SystemExit
 
-    if localbase:
-        incdirs = [localbase+d for d in origincdirs]
-        libdirs = [localbase+d for d in origlibdirs]
-    else:
-        incdirs = []
-        libdirs = []
+    incdirs = []
+    libdirs = []
     incdirs += ["/usr"+d for d in origincdirs]
     libdirs += ["/usr"+d for d in origlibdirs]
     incdirs += ["/usr/local"+d for d in origincdirs]
     libdirs += ["/usr/local"+d for d in origlibdirs]
+    if localbase:
+        incdirs = [localbase+d for d in origincdirs]
+        libdirs = [localbase+d for d in origlibdirs]
 
     for arg in DEPS[0].cflags.split():
         if arg[:2] == '-I':
